numba_cuda/numba/cuda/cuda_paths.py

```diff
@@ -256,3 +258,66 @@ def get_debian_pkg_libdevice():
     if not os.path.exists(pkg_libdevice_location):
         return None
     return pkg_libdevice_location
+
+
+def get_current_cuda_target_name():
+    """Determine conda's CTK target folder based on system and machine arch.
+
+    CTK's conda package delivers headers based on its architecture type. For example,
+    `x86_64` machine places header under `$CONDA_PREFIX/targets/x86_64-linux`, and
+    `aarch64` places under `$CONDA_PREFIX/targets/sbsa-linux`. Read more about the
+    nuances at cudart's conda feedstock:
+    https://github.com/conda-forge/cuda-cudart-feedstock/blob/main/recipe/meta.yaml#L8-L11  # noqa: E501
+    """
+    system = platform.system()
+    machine = platform.machine()
+
+    if system == "Linux":
+        arch_to_targets = {
+            'x86_64': 'x86_64-linux',
+            'aarch64': 'sbsa-linux'
+        }
+        return arch_to_targets.get(machine)
+
```

Comment:
Are we missing the Windows (`"x64"`?) case here?

Reply:
Yes, it should be this:
```python
    if system == "Windows":
        arch_to_targets = {
            'AMD64': 'x64'
        }
        return arch_to_targets.get(machine)
```

Reply:
Done in https://github.com/NVIDIA/numba-cuda/pull/17/commits/9b6dda9dd16ec678c303c9fcb733de80e5a4c628